docs/UserGuide.md

```diff
@@ -420,8 +427,9 @@ ModuleList's data being saved as `[JAR file location]/data/modulelist.json`
 Advanced users are welcome to update the data directly by editing those data files.
 
 <div markdown="span" class="alert alert-warning"> :exclamation: **Caution:**
-If your changes to the data file makes its format invalid, TAilor will discard all data and start
-with an empty data file at the next run.
+If your changes to the data file makes its format invalid, TAilor will discard data in all 3 files and start
+with no data at the next run. It is suggested that users manually backup copies of these files before directly editing
```

Comment:
We did discuss adding possible scenarios that could corrupt the data of a particular file so that the user knows when it is ideal to manually back up. Do you want to go ahead and add some of those scenarios too? Can just follow whatever our PE-D reviewer did for one example